examples/analytics-notification.amp.html

```diff
@@ -66,8 +66,8 @@
   <amp-user-notification
       layout=nodisplay
       id="amp-user-notification1"
-      data-show-if-href="http://localhost:8000/api/show?timestamp=TIMESTAMP"
```

Comment:
If we want to really force resources over HTTPS, then we should not allow "//" as well. Otherwise document on HTTP would still load non-HTTPS resources ?